File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        d = {}
        ans =[]
        for num in nums:
            if num not in d:
                d[num] = 1
            else:
                d[num] += 1
        
        for i in range(k):
            maxFreq = 0
            res = 0
            for key in d:
                if d[key] > maxFreq:
                    maxFreq = d[key]
                    res = key
            ans.append(res)
            del d[res]

        return ans

File: Data Structures & Algorithms/concatenation-of-array/submission-0.java
class Solution {
    public int[] getConcatenation(int[] nums) {
        int n = nums.length;
        int[] ans = new int[2*n];
        int i = 0, j = n;
        while (i < n) {
            ans[i] = nums[i];
            ans[j] = nums[i];
            i++;
            j++;
        }
        return ans;
    }
}